src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/PersonModal/PersonPhoneNumber/PersonPhoneNumber.tsx

```diff
@@ -150,6 +153,7 @@ export const PersonPhoneNumber: React.FC<PersonPhoneNumberProps> = ({
                             }}
                             fullWidth
                           >
+                            <MenuItem selected value=""></MenuItem>
```

Comment:
So this is making the empty menu item be the default right?

Reply:
Yep, since the default value I set is ```null``` it was throwing warnings in the console about not matching an option in the ```Select```. And the value of a ```Select``` cannot be ```null``` so I have to change it to an empty string. And to get rid of that warning I just added an option with the value of an empty string.